src/wp-includes/media.php

```diff
@@ -1969,6 +1969,7 @@ function wp_filter_content_tags( $content, $context = null ) {
  * @return string Converted `img` tag with optimization attributes added.
  */
 function wp_img_tag_add_loading_optimization_attrs( $image, $context ) {
+	$src               = preg_match( '/< *img[^>]*src *= *["\']?([^"\']*)/i', $image, $matche_src ) ? $matche_src[1] : null;
```

Comment:
can this regex exclude the img prefix part so it is closer to the ones below?

Reply:
Yes we can exclude img, thanks